packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -12350,7 +12350,7 @@ export default{
 			msw.use(handler);
 		}
 
-		it("should log open-beta warning when deploying a workflow", async () => {
+		it("should not log open-beta warning when deploying a workflow", async () => {
```

Comment:
```suggestion
		it("should deploy a workflow", async () => {
```

Reply:
IMHO it doesn't make much sense to test specifically for the absence of the open beta warning